### DISCUSSION TITLE
Installing Brew with Rosetta2 fails

### DISCUSSION THREAD

**Discussion by ecwio** [Homebrew/discussions#260] [Getting started] [answered]
Trying to install homebrew with the new apple M1 chip

`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

but I get error 

`Failed to link all completions, docs and manpages:
  Permission denied @ rb_file_s_symlink - (../../../Homebrew/completions/zsh/_brew, /usr/local/share/zsh/site-functions/_brew)
Failed during: /usr/local/bin/brew update --force`

**Comment by SMillerDev** [comment, marked as answer]
You installed something that changed the permission on that location, `/usr/local/bin/brew doctor` should tell you what directories are wrong exactly.
